src/pages/commitqueue/CommitQueueCard.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Subtitle, Body } from "@leafygreen-ui/typography";
+import { uiColors } from "@leafygreen-ui/palette";
+import Button from "@leafygreen-ui/button";
+import { CodeChangeModule } from "pages/commitqueue/codeChangesModule/CodeChangesModule";
+import { StyledRouterLink } from "components/styles/StyledLink";
+import { ModuleCodeChanges } from "types/patch";
+import { paths } from "constants/routes";
+import { format } from "date-fns";
+const FORMAT_STR = "MM/dd/yy' at 'hh:mm:ss' 'aa";
+
+interface Props {
+  index: number;
+  title: string;
+  author: string;
+  commitTime: Date;
+  patchId: string;
+  moduleCodeChanges: [ModuleCodeChanges];
+}
+const { blue, gray } = uiColors;
+
+export const CommitQueueCard: React.FC<Props> = ({
+  index,
+  title,
+  author,
+  commitTime,
+  patchId,
+  moduleCodeChanges,
+}) => {
+  return (
+    <Card data-cy="commit-queue-card">
+      <Subtitle>{index}.</Subtitle>
+      <CommitQueueCardGrid>
+        <CommitInfo>
+          <CardTitle to={`${paths.patch}/${patchId}`}>{title}</CardTitle>
+          <CardMetaData>
+            By <b>{author}</b> on {format(new Date(commitTime), FORMAT_STR)}
+          </CardMetaData>
+          <div>
+            {moduleCodeChanges.map((moduleCodeChange) => (
+              <CodeChangeModule
+                key={moduleCodeChange.rawLink}
+                moduleCodeChange={moduleCodeChange}
+              />
+            ))}
+          </div>
```

Comment:
prefer a React.fragment where unstyled div is used to contain content. Shorthand for which is `<>{content}</>`